todo-frontend/src/components/App.vue

```diff
@@ -2,17 +2,19 @@
   <div id="app">
     <div style="margin-bottom: 200px;">
       <ul>
-        <div v-if="checkToken() == false" >
+        
+
+        <div v-if="this.provera() != true">
           <li>
             <router-link to="/Login"> Login </router-link>
           </li>
           <li>
             <router-link to="/register"> Register </router-link>
           </li>
         </div>
-        <div v-if="checkToken()">
+        <div v-if="this.provera() == true">
```

Comment:
1. bez this u template-u
2. nema potrebe da eksplicitno pises == true, sam ce to evaluirati kao true
3. necemo srpski
4. kad "provera" ode u computed ovde ce ostati samo v-if="provera"